libs/ethereum.red

```diff
@@ -28,7 +27,7 @@ eth: context [
 		mul256 n GWei-ratio
 	]
 
-	pad64: function [data [string! binary!]][
+	pad64: func [data [string! binary!] /local c len n][
```

Comment:
Why this change?

Reply:
Consistency and speed and new comers should better use `func` to avoid headaches later. I remember you saying to have been bitten by this. Your call, it looks nicer with `function`

Reply:
Right, but we are not "newcomers", so `function` is what we want to use there. ;-) For performance, unless your code is creating millions of functions, there won't be any impact on apps performance.

Reply:
I didn't mean to alter the code, I just thought it was a leftover from quick development because you guys are so busy. But I appreciate your comment, I will revert it and never do it again! :-)